src/gen_rpc_client.erl

```diff
@@ -389,3 +422,17 @@ merge_timeout_values(SRecvTO, undefined, _SSendTO, USendTO) ->
     {SRecvTO, USendTO};
 merge_timeout_values(_SRecvTO, URecvTO, _SSendTO, USendTO) ->
     {URecvTO, USendTO}.
+
+%% Transform result for safe_eval_everywhere to look like multicall
+transform_result(Result, Nodes) ->
```

Comment:
I think this part can be improved. We only care about badrpc/badtcp so why not a list comprehension with filtering the bad nodes like `{_, _} = X` ?